dts/bindings/display/samsung,ks0108.yaml

```diff
@@ -0,0 +1,84 @@
+description: KS0108 128x64 display controller
+
+compatible: "samsung,ks0108"
+
+properties:
+    db0-gpios:
+      type: phandle-array
+      required: true
+      description: DB0 pin.
```

Comment:
Since these are all required, why not make a `db-gpios` that has length 8?

Then you can use `DT_FOREACH_PROP_ELEM` to construct a `gpio_dt_spec` for each element.